packages/gasket-utils/lib/apply-config-overrides.js

```diff
@@ -1,49 +1,71 @@
 const path = require('path');
+// @ts-ignore
 const defaultsDeep = require('lodash.defaultsdeep');
 const tryRequire = require('./try-require');
 
+// @ts-ignore
 const debug = require('diagnostics')('gasket:utils');
 
 /**
  * Normalize the config by applying any overrides for environments, commands,
  * or local-only config file.
- *
- * @param {object} config - Target config to be normalized
- * @param {object} context - Context for applying overrides
- * @param {string} context.env - Name of environment
- * @param {string} [context.commandId] - Name of command
- * @param {string} [context.root] - Project root; required if using localeFile
- * @param {string} [context.localFile] - Optional file to load relative to gasket root
- * @returns {object} config
+ * @type {import('./index').applyConfigOverrides}
  */
-function applyConfigOverrides(config, { env = '', commandId, root, localFile }) {
+function applyConfigOverrides(
+  config,
+  { env = '', commandId, root, localFile }
+) {
   return defaultsDeep(
     {},
     ...getPotentialConfigs({ config, env, commandId, root, localFile })
   );
 }
 
+/**
+ * Generator function to yield potential configurations
+ * @type {import('./index').getPotentialConfigs}
+ */
 function *getPotentialConfigs({ config, env, commandId, root, localFile }) {
   // Separate environment-specific config from other config
+  // @ts-ignore
```

Comment:
Why all the `@ts-ignores` in this function?

Reply:
This was included by mistake. Just pushed the update.